docs/rpc/rpc-spans.md

```diff
@@ -84,13 +84,13 @@ Examples of span names:
 <!-- semconv rpc(full) -->
```

Comment:
I think we need to get rid of common attributes table here similarly to HTTP

Reply:
Agreed - I'll open an issue / tackle this outside of this PR.

Reply:
https://github.com/open-telemetry/semantic-conventions/issues/959